deploy/crds/quickstart-bootable-jar-cr.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: wildfly.org/v1alpha1
+kind: WildFlyServer
+metadata:
+  name: quickstart-bootable-jar
+spec:
+  applicationImage: "quay.io/wildfly-quickstarts/wildfly-operator-quickstart:bootable-21.0"
```

Comment:
@jmesnil This image does not exist yet, you could tag and push to wildfly-quickstarts the onw I've been using quay.io/yborgess/jaxrs-demo:1.0. 

The quay.io/wildfly-quickstarts/wildfly-operator-quickstart:bootable-21.0 is also needed for the test.

Reply:
done, pushed to quay.io/wildfly-quickstarts/wildfly-operator-quickstart:bootable-21.0